packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -613,7 +613,51 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 	/** @inheritDoc */
 	@rethrowFileSystemError('Could not move "{path}"')
 	mv(fromPath: string, toPath: string) {
-		this[__private__dont__use].FS.rename(fromPath, toPath);
+		const mv = (fromPath: string, toPath: string) => {
```

Comment:
A small nitpick: it seems unnecessary to create an internal function `mv` here, which has the same signature as the method and is called immediately after creation with the same arguments.